tests/debugging/test_debugger.py

```diff
@@ -699,3 +701,18 @@ def test_debugger_function_probe_duration(duration):
 
         (snapshot,) = d.test_queue
         assert 0.9 * duration <= snapshot.duration <= 2.5 * duration, snapshot
+
+
+def test_debugger_run_module():
```

Comment:
Do we want to also check the debugger snapshot?

Reply:
The emitted snapshot is checked in the `target.py` source, where essentially the testing logic is, if this is what you meant?